lua/lspconfig/server_configurations/drools_lsp.lua

```diff
@@ -0,0 +1,119 @@
+local util = require 'lspconfig.util'
+
+local get = function(config, section, key)
+  if config.settings then
+    if config.settings[section] then
+      return config.settings[section][key]
+    end
+  end
+  return nil
+end
+
+local get_java_bin = function()
+  local java_bin = vim.env.JAVA_HOME and util.path.join(vim.env.JAVA_HOME, 'bin', 'java') or 'java'
+  if vim.fn.has 'win32' == 1 then
+    java_bin = java_bin .. '.exe'
+  end
+  return java_bin
+end
+
+local get_drools_jar = function()
+  local drools_jar = 'drools-lsp-server-jar-with-dependencies.jar'
+  -- support for https://github.com/williamboman/mason.nvim
+  local mason_exists, mason_path = pcall(require, 'mason-core.path')
```

Comment:
this repo just provide a basic server config it would be better not add any thirdpart plugin relate code.

Reply:
@glepnir Okay, I can change this too, and update the docs section to match.

Reply:
@glepnir Done. I can recommend mason.nvim users to do this:
```lua
require('lspconfig').drools_lsp.setup {
  settings = {
    drools = {
      jar = (function()
        local mason_path = require('mason-core.path')
        local mason_pkg = mason_path.package_prefix 'drools-lsp'
        return mason_path.concat { mason_pkg, 'drools-lsp-server-jar-with-dependencies.jar' }
    end)(),
    },
  },
}
```

Reply:
this is most better :)